contexts/v1.jsonld

```diff
@@ -16,54 +16,39 @@
       }
     },
 
-    "RevocationList2021": {
+    "StatusList2021": {
       "@id":
-        "https://w3id.org/vc/status-list#RevocationList2021",
+        "https://w3id.org/vc/status-list#StatusList2021",
       "@context": {
         "@protected": true,
 
         "id": "@id",
         "type": "@type",
 
+        "statusPurpose":
+          "https://w3id.org/vc/status-list#statusPurpose",
         "encodedList": "https://w3id.org/vc/status-list#encodedList"
       }
     },
 
-    "RevocationList2021Status": {
+    "StatusListEntry2021": {
```

Comment:
It seems like this should be `StatusList2021Entry`, i.e., everything should be `StatusList2021X`.

Reply:
Same comment as https://github.com/w3c-ccg/vc-status-list-2021/pull/24/files#r825341914.

Reply:
Would be good to get some more opinions on this as well. /cc @OR13 @mprorock @mkhraisha @tplooker @mavarley @TallTed @peacekeeper

Reply:
Fixed in 01a681de2d2c2b91fc13d5ed6b272ea27254abe4.